docs/git/getting-started.md

```diff
@@ -0,0 +1,176 @@
+# Git Getting Started
+
+The following is a getting started guide for using [git] while developing
+Cardano SL.
+
+Its probably best to avoid the use of GUIs that try to make the use of [git]
+easier. Git is a complex tool with many sub commands and each sub command has
```

Comment:
Apart from magit (emacs git mode) - magit is awesome :-)

Reply:
or fugitive in vim, but hey, if one is using vim or emacs one probably doesn't need this guide.